lib/zold/commands/fetch.rb

```diff
@@ -69,12 +71,16 @@ def run(args = [])
         o.string '--network',
           'The name of the network we work in',
           default: 'test'
+        o.integer '--threads',
+          "How many threads to use for fetching wallets (default: #{[Concurrent.processor_count / 2, 2].max})",
+          default: [Concurrent.processor_count / 2, 2].max
         o.bool '--help', 'Print instructions'
       end
       mine = Args.new(opts, @log).take || return
-      (mine.empty? ? @wallets.all : mine.map { |i| Id.new(i) }).each do |id|
+      Parallel.map((mine.empty? ? @wallets.all : mine.map { |i| Id.new(i) }), in_threads: opts[:threads] ) { |id|
         fetch(id, Copies.new(File.join(@copies, id)), opts)
-      end
+        @log.debug("Worker: #{Parallel.worker_number} has fetched wallet #{id}")
+      }
```

Comment:
@emilianodellacasa did you try to run rubocop? I believe it will complain. you should use `begin/end` if there are multiple lines

Reply:
@yegor256 You are right, fixed